README.md

```diff
@@ -57,5 +57,5 @@ All files are generated as static html pages in `/build` folder.
 
 ## Publish
 
-See [pay-team-manual/publish-api-docs](https://pay-team-manual.cloudapps.digital/release-processes/publish-public-api-docs/)
+See [pay-team-manual/publish-api-docs](https://pay-team-manual.cloudapps.digital/manual/publish-public-api-docs.html)

```

Comment:
This is now `https://pay-team-manual.cloudapps.digital/manual/release-processes/publish-public-api-docs.html`